tests/lifecycle/lifeparameters/parameters.go

```diff
@@ -19,37 +19,19 @@ var (
 		testPodLabelPrefixName: testPodLabelValue,
 		"app":                  "test",
 	}
-	PreStopCommand          = []string{"/bin/sh", "-c", "killall -0 tail"}
-	LifecycleTestSuiteName  = "lifecycle"
-	SkipAllButShutdownRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
-		" lifecycle-pod-termination-grace-period lifecycle-pod-owner-type" +
-		" lifecycle-pod-recreation lifecycle-deployment-scaling lifecycle-image-pull-policy"
-
-	SkipAllButScalingRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
-		" lifecycle-pod-termination-grace-period lifecycle-pod-owner-type" +
-		" lifecycle-pod-recreation lifecycle-container-shutdown lifecycle-image-pull-policy"
-
-	SkipAllButTerminationGracePeriodRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
-		" lifecycle-deployment-scaling lifecycle-pod-owner-type" +
-		" lifecycle-pod-recreation lifecycle-container-shutdown lifecycle-image-pull-policy"
-
-	SkipAllButPodOwnerTypeRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
-		" lifecycle-deployment-scaling lifecycle-pod-termination-grace-period" +
-		" lifecycle-pod-recreation lifecycle-container-shutdown lifecycle-image-pull-policy"
-
-	SkipAllButPodRecreationRegex = "lifecycle-pod-high-availability lifecycle-pod-scheduling" +
-		" lifecycle-deployment-scaling lifecycle-pod-termination-grace-period" +
-		" lifecycle-pod-owner-type lifecycle-container-shutdown lifecycle-image-pull-policy"
-
-	SkipAllButPodHighAvailabilityRegex = "lifecycle-pod-recreation  lifecycle-pod-scheduling" +
-		" lifecycle-deployment-scaling lifecycle-pod-termination-grace-period" +
-		" lifecycle-pod-owner-type lifecycle-container-shutdown lifecycle-image-pull-policy"
-
-	ShutdownDefaultName               = "lifecycle lifecycle-container-shutdown"
-	DeploymentScalingDefaultName      = "lifecycle lifecycle-deployment-scaling"
-	TerminationGracePeriodDefaultName = "lifecycle lifecycle-pod-termination-grace-period"
-	PodOwnerTypeDefaultName           = "lifecycle lifecycle-pod-owner-type"
-	PodRecreationDefaultName          = "lifecycle lifecycle-pod-recreation"
-	PodHighAvailabilityDefaultName    = "lifecycle CNF is designed in high availability mode" +
-		"lifecycle-pod-high-availability"
+	PreStopCommand         = []string{"/bin/sh", "-c", "killall -0 tail"}
+	LifecycleTestSuiteName = "lifecycle"
+
+	SkipsTcsSlice = []string{"lifecycle-pod-recreation", "lifecycle-pod-scheduling",
```

Comment:
You don't need the "Slice" suffix. Also, you're using it as a skip tc list, but this is just the whole list of TCs in the tnf test suite, so, to me, a better name for this name would be just "lifecycleTnfTestCases", or just "TnfTestCases".

Reply:
Much better name, thank you, renaming it.